frb_codegen/src/library/codegen/parser/mod.rs

```diff
@@ -34,7 +34,7 @@ fn parse_inner(
     let dumper_mir = dumper.with_content(Mir);
 
     let pb = progress_bar_pack.parse_hir_raw.start();
-    let hir_raw = hir::raw::parse(&config.hir, dumper)?;
+    let hir_raw = hir::raw::parse(&config.hir, dumper, config.features.as_deref())?;
```

Comment:
... then we do not need to add this extra arg, since it is already in `config.hir`